tests/raw_query/tests.py

```diff
@@ -247,81 +249,104 @@ def test_query_representation(self):
         """
         Test representation of raw query with parameters
         """
-        query = "SELECT * FROM raw_query_author WHERE last_name = %(last)s"
-        qset = Author.objects.raw(query, {"last": "foo"})
+        query = [{"$match": {"last_name": "%(last)s" % {"last": "foo"}}}]
+        qset = Author.objects.raw_mql(query)
         self.assertEqual(
             repr(qset),
-            "<RawQuerySet: SELECT * FROM raw_query_author WHERE last_name = foo>",
+            "<MongoRawQuerySet: [{'$match': {'last_name': 'foo'}}]>",
         )
         self.assertEqual(
             repr(qset.query),
-            "<RawQuery: SELECT * FROM raw_query_author WHERE last_name = foo>",
+            "<MongoRawQuery: [{'$match': {'last_name': 'foo'}}]>",
         )
 
-        query = "SELECT * FROM raw_query_author WHERE last_name = %s"
-        qset = Author.objects.raw(query, {"foo"})
+        query = [{"$match": {"last_name": "%s" % "foo"}}]
+        qset = Author.objects.raw_mql(query)
         self.assertEqual(
             repr(qset),
-            "<RawQuerySet: SELECT * FROM raw_query_author WHERE last_name = foo>",
+            "<MongoRawQuerySet: [{'$match': {'last_name': 'foo'}}]>",
         )
         self.assertEqual(
             repr(qset.query),
-            "<RawQuery: SELECT * FROM raw_query_author WHERE last_name = foo>",
+            "<MongoRawQuery: [{'$match': {'last_name': 'foo'}}]>",
```

Comment:
I think these tests can be skipped since we ultimately don't support the params field

Reply:
We do need one test for `__repr__()` but we don't need to test this redundantly or with a query that uses string interpolation. Let's continue to edit this file here (so we can see the diff against the `raw_query` tests, but with the ultimate goal of putting this test app in the django-mongodb tests directory rather than merging these changes. So, for example, for tests that aren't applicable, remove them in the PR instead of adding skips.

Reply:
E.g. `raw_query_` ?